services/core-web/src/components/common/DocumentTable.tsx

```diff
@@ -348,42 +379,21 @@ export const DocumentTable = ({
     let element = (
       <CoreTable
         columns={columns}
-        {...(enableBulkActions
-          ? {
-              rowSelection: {
-                type: "checkbox",
-                ...rowSelectionObject,
-              },
-            }
-          : {})}
         dataSource={documents}
+        {...bulkActionsProps}
         {...minimalProps}
       />
     );
 
     if (showVersionHistory) {
       element = (
-        <div>
-          <CoreTable
-            condition={isLoaded}
-            dataSource={documents}
-            columns={columns}
-            {...(enableBulkActions
-              ? {
-                  rowSelection: {
-                    type: "checkbox",
-                    ...rowSelectionObject,
-                  },
-                }
-              : {})}
-            expandProps={{
-              childrenColumnName: "versions",
-              matchChildColumnsToParent: true,
-              recordDescription: "version history",
-              rowExpandable: (record) => record.number_prev_versions > 0,
-            }}
-          />
-        </div>
+        <CoreTable
```

Comment:
I think we can just return a single instance.
like 
```
<CoreTable
condition
dataSource
columns
...bulkActions
...versions
...minimal
/>
```

The condition prop I don't think was removed on purpose, and minimalProps is going to be null anyway unless we explicitly pass in that we want it to be minimal. (If there's an issue with minimalProps in combination with versionProps, I think we can worry about it when we set both on the same table at the same time).